integration-tests/maven/src/test/resources/projects/custom-manifest-section/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I think it should be possible to test this using QuarkusProdModeTest rather than a maven integration test. QuarkusProdModeTest is much more lightweight and a lot faster to run, so should be used where possible.

Reply:
@stuartwdouglas Where is the best place to put the test? Should I keep the test in integration-tests/maven or move them into integration-tests/main?

Reply:
The `maven` one, please.